CONTRIBUTING.MD

```diff
@@ -157,7 +157,7 @@ the Apache Spark contributor guidelines, and the Kubernetes contributor guide.
 [github]: https://lab.github.com/
 [contributor license agreement]: https://cla-assistant.io/camunda-cloud/camunda-cloud-documentation
 [code style guidelines]: https://github.com/camunda-cloud/zeebe/wiki/Code-Style
-[testing best practices]: https://docs.camunda.io/docs/apis-clients/java-client/zeebe-process-test/
+[testing best practices]: https://docs.camunda.io/docs/apis-tools/java-client/zeebe-process-test/
```

Comment:
```suggestion
[testing best practices]: https://docs.camunda.io/docs/apis-clients/java-client/zeebe-process-test/
```

blocking: I don't think this URL will resolve even after this deploys, because it is a v8.1 URL and v8.1 is still located at `apis-clients`.